test/shlibloadtest.c

```diff
@@ -117,6 +117,7 @@ static int test_lib(void)
         if (!TEST_true(shlib_load(path_crypto, &cryptolib))
                 || !TEST_true(shlib_load(path_ssl, &ssllib)))
             goto end;
+        break;
     case SSL_FIRST:
         if (!TEST_true(shlib_load(path_ssl, &ssllib))
                 || !TEST_true(shlib_load(path_crypto, &cryptolib)))
```

Comment:
I am not sure if the `|| !TEST` thing should be aligned to the `!` above. But it seems doing so will help make the code at a low risk to exceed 80 boundary.